### PR TITLE
Fixes wrong headline level of tracestate Header Field Values

### LINKED ISSUE #403: Fix numbering/level in rendered spec
In the final spec, the section *tracestate Header Field Values* should be one level up https://www.w3.org/TR/trace-context/#tracestate-header-field-values

### PR DESCRIPTION
**Pull request by danielkhan**
closes #403

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/trace-context/pull/404.html" title="Last updated on Jun 2, 2020, 3:56 PM UTC (88f4db3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/trace-context/404/85789a7...88f4db3.html" title="Last updated on Jun 2, 2020, 3:56 PM UTC (88f4db3)">Diff</a>

### CODE REVIEW

**Review by dyladan — APPROVED** [review]

**Review by mwear — APPROVED** [review]

**Review by andrewhsu — APPROVED** [review]
LGTM

File: spec/20-http_request_header_format.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ In order to increase interoperability across multiple protocols and encourage su
 
 Vendors MUST expect the header name in any case (upper, lower, mixed), and SHOULD send the header name in lowercase.
 
-##### tracestate Header Field Values
+### tracestate Header Field Values
```
Comment (arminru) [inline spec/20-http_request_header_format.md:238]:
```suggestion
### Header Values
```
redundant, same for `## Traceparent > ### traceparent Header Field Values` above

**Review by arminru — APPROVED** [review]

File: spec/20-http_request_header_format.md [hunk 2]
```diff
@@ -235,7 +235,7 @@ In order to increase interoperability across multiple protocols and encourage su
 
 Vendors MUST expect the header name in any case (upper, lower, mixed), and SHOULD send the header name in lowercase.
 
-##### tracestate Header Field Values
+### Header Values
```
Comment (SergeyKanzhelev) [inline spec/20-http_request_header_format.md]: this header will be harder to link to directly with anchor links

**Review by SergeyKanzhelev — CHANGES_REQUESTED** [review]
Can you please split PR into two: fixing indents and rename of a title? Perhaps better title can be found that is both - unique in a document and clearer.

File: spec/20-http_request_header_format.md (hunk 2 above)
Comment (danielkhan) [reply spec/20-http_request_header_format.md]: That would be the same for `Header Name` - it isn't unique either and the document processor is smart enough to create 
* https://www.w3.org/TR/trace-context/#header-name
and
* https://www.w3.org/TR/trace-context/#header-name-0
to distinguish between same headlines.

**Comment by danielkhan** [comment]
> Can you please split PR into two: fixing indents and rename of a title? Perhaps better title can be found that is both - unique in a document and clearer.

See my reply to your comment. To keep this simple, I am reverting the change of the header names here and will create a new PR later.

**Review by SergeyKanzhelev — APPROVED** [review]

**Pull request merged**